src/test/java/org/broadinstitute/hellbender/tools/picard/analysis/directed/CollectJumpingLibraryMetricsIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.hellbender.tools.picard.analysis.directed;
+
+/**
+ * Created by dkling on 9/1/15.
+ */
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+
+    public final class CollectJumpingLibraryMetricsIntegrationTest extends CommandLineProgramTest {
+
+        private static final File TEST_DATA_PATH = new File(getTestDataDir(), "picard/analysis/directed.CollectJumpingLibraryMetrics/");
```

Comment:
the directory should be called `picard/analysis/directed/CollectJumpingLibraryMetrics` not 
`picard/analysis/directed.CollectJumpingLibraryMetrics`